business_objects/admin_message.py

```diff
@@ -0,0 +1,54 @@
+from typing import List
+from submodules.model.models import AdminMessage
+from . import general
+from ..session import session
+from datetime import datetime
+
+
+def get(message_id: str) -> AdminMessage:
+    return session.query(AdminMessage).filter(AdminMessage.id == message_id).first()
+
+
+def get_all() -> List[AdminMessage]:
+    return session.query(AdminMessage).filter().limit(100).all()
+
+
+def get_all_active() -> List[AdminMessage]:
+    return (
+        session.query(AdminMessage)
+        .filter(AdminMessage.archived == False)
+        .limit(100)
```

Comment:
all_active suggests all however there is a limit included. To make it easier understandable i'd give it a parameter limit that defaults to 100 so the dev can decide if necessary to exclude a limit (plays a bit into a comment from gateway)

Reply:
done